src/main/java/com/truelayer/java/ITrueLayerClient.java

```diff
@@ -33,4 +37,13 @@
      * @return a utility to build a Hosted Payment Page URL.
      */
     IHostedPaymentPageLinkBuilder hpp();
+
+    /**
+     * Utility to submit payment returns parameters.
+     * @param request a submit payment return parameters payload
+     * @return the response of the <i>Submit payment returns parameters</i> operation
+     * @see <a href="https://docs.truelayer.com/reference/submit-payments-provider-return-parameters"><i>Submit payments return parameters</i> API reference</a>
+     */
+    CompletableFuture<ApiResponse<SubmitPaymentReturnParametersResponse>> submitPaymentReturnParameters(
+            SubmitPaymentReturnParametersRequest request);
```

Comment:
Is this a matter of improving the usability for the SDK user? So that they can do something like:
```
myTLClient.submitPaymentReturnsParameters(...)
```
instead of:
```
myTLClient.some_akward_handler_name_like_common().submitPaymentReturnsParameters(...)
```